### DISCUSSION TITLE
mqtt clean session flag

### DISCUSSION THREAD

**Discussion by borti-16** [tbnobody/OpenDTU#1297] [Q&A]
Hi,
I am looking for the standard setting of the clean session flag. The flag can not be set on the mqtt GUI and I could not find a fixed setting in the code (which might be through my limitation there though). Does anyone know the setting the system is publishing with? and possibly also how to change it?

**Comment by tbnobody** [comment]
currently not possible. I will add this config flag soon.

**Reply by borti-16** [reply]
Thank you :)
